.github/workflows/nodejs.yml

```diff
@@ -24,8 +24,12 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - run: npm ci
-    - run: npm run build --if-present
-    - run: npm test
+    - name: Test
+      run: |
+        npm ci
+        npm run build --if-present
+        npm start
```

Comment:
I think the build is getting stuck as this command isn't running in the background, we may need to look at `pm2/nodemon`?

Reply:
Yeah, I just noticed that and I've tweaked that to fix it.